frontend/test/cypress/integration/product/pro_user.cy.ts

```diff
@@ -2,13 +2,33 @@ describe('Pro user test', () => {
    it('will give pro users a discount', () => {
       cy.loadProductPageByPath('/products/repair-business-toolkit');
 
+      // Assert product title is visible
+      cy.findByTestId('product-title')
+         .should('be.visible')
+         .invoke('text')
+         .then((productTitle) => {
+            expect(productTitle).to.equal('Repair Business Toolkit');
+         });
+
+      // Assert product sku is visible
+      cy.findByTestId('product-sku')
+         .should('be.visible')
+         .invoke('text')
+         .then((productSku) => {
+            expect(productSku).to.match(/IF\d*\-\d*/g);
+         });
```

Comment:
This doesn't really have to do with a pro user (the name of this file). How about we pull this out into a `product_page.cy.ts` file?

Reply:
We can create a new test, but it will create additional few seconds to load up the same product page. 
I added it in this test since we already visit a product page so we can save up the time for tests to run.

Reply:
I suppose it's only `2-4` seconds, [ref](https://github.com/iFixit/react-commerce/actions/runs/3500832448/jobs/5863890797#step:6:205).

Reply:
Yeah, I guess it doesn't seem so bad right now, but if we start adding a ton of tests then I think we will want separate logic tested in separate places.

This is an extreme example, but if I wanted to check if campaigns were tested but the test code was in a file named `answersTest`, I wouldn't think campaigns were tested. Let's split this up even if it adds a bit of time.

Reply:
Sounds good! Split them in https://github.com/iFixit/react-commerce/pull/1050/commits/5d3fd0af93406ec51159c34a033b10c77cd39b8d.

Reply:
Thank you! Looks great!